back/src/Controller/IoSocketController.ts

```diff
@@ -233,30 +236,46 @@ export class IoSocketController {
             },
             /* Handlers */
             open: (ws) => {
-                // Let's join the room
-                const client = this.initClient(ws); //todo: into the upgrade instead?
-                socketManager.handleJoinRoom(client);
-                resetPing(client);
+                (async () => {
+                    // Let's join the room
+                    const client = this.initClient(ws); //todo: into the upgrade instead?
 
-                //get data information and shwo messages
-                adminApi.fetchMemberDataByUuid(client.userUuid).then((res: FetchMemberDataByUuidResponse) => {
-                    if (!res.messages) {
-                        return;
+                    const room = socketManager.getRoomById(client.roomId);
+                    if (room && room.isFull) {
+                        socketManager.emitCloseMessage(client, 302);
```

Comment:
Shouldn't we close the socket after emitting the close message?

Reply:
J'ai un code plus loin qui permet d'arrêter la connexion si la room est pleine : 
```
if (room && room.isFull) {
                    return;
                }
```